runtime/tests/js/ipfs_retrieval_tests.js

```diff
@@ -2,12 +2,13 @@ import { test } from "zinnia:test";
 import { assertEquals, assertMatch, assertRejects, AssertionError } from "zinnia:assert";
 
 const TEST_CID = "bafkreih25dih6ug3xtj73vswccw423b56ilrwmnos4cbwhrceudopdp5sq";
+const RETRIEVE_FROM_OUR_FRISBII = "?protocol=http&providers=/dns4/frisbii.fly.dev/https";
```

Comment:
I'm having issues with this name. I propose changing it to something like `SPACE_MERIDIAN_FRISBII_INSTANCE` or `FRISBII_INSTANCE_QUERY_PARAMS`?